src/widget/wbeatspinbox.cpp

```diff
@@ -295,6 +307,36 @@ void WBeatSpinBox::keyPressEvent(QKeyEvent* pEvent) {
         ControlObject::set(ConfigKey("[Library]", "refocus_prev_widget"), 1);
         return;
     }
+    if (this->size() == sizeSpinBox) {
+        if (pEvent->key() == Qt::Key_Right && pEvent->modifiers() == Qt::ControlModifier) {
+            beatJumpOneForward.set(beatJumpSize.get());
```

Comment:
Please check what `beatJumpSize.get()` returns.
And which values make sense for [`beatjump_N_forward`](https://github.com/mixxxdj/mixxx/blob/b1967eaeabd8012ec7d51bce7e8c59e18011d39a/src/engine/controls/loopingcontrol.cpp#L1945) and `beatjump_forward/_backward`.

Reply:
`this->` is redundant btw.